_posts/2019-05-08-BA-and-Beyond.md

```diff
@@ -0,0 +1,124 @@
+She is active in the BA professional community in the UK, attending and regularly speaking at local and national events and conferences. 
+
+Christina's talk was purely soft skills related and depicts the sort of behaviours shown by business analysts when they face difficult situations.
+Some typical BA behavioural mistakes are triggered by the way they see themselves in the team. 
+For instance, a BA could feel to not be part of the team: there are de facto less analysts than developers in a team which can lead to an «Us VS Them» mentality.
+It is also sometimes difficult to build a BA community and the BA role sometimes being sandwiched between IT and the business might lead to negative self perceptions such as «I am just a BA» or «Neither the PO nor the developers understand me».
+
+<img alt="bad-ba-behaviours" src="{{ '/img/ba-and-beyond-2019/bad-ba-behaviours.jpg' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 500px;">
+
+Understanding that the BA is fully part of the team and is not the only one accountable for the communication between IT and Business, leads to better collaboration and is a great step towards agility, allowing all team members to be more confident and positive.
+Whatever your role in a project team; collaboration, transparency, confidence - in yourself and in your team mates - as well as clear communication are key factors to succeed together.
+
+Another difficulty in the BA role is having the right level of detail. 
+Indeed it is sometimes difficult to have the right level of focus on the right things. 
+Moreover, being a good BA is also about sharing knowledge, tools and techniques. 
+Talking with other analysts and contributing to the practice are important aspects of the job.
+
+Of course all those behaviours and feelings don’t just apply to business analysts but also to every team role: scrum master, developer, product owner, tester.
+This talk gave us the opportunity to identify and name some feelings we can face as business analysts while giving us the keys to find solutions to the difficult situations we could face.
+
+<img alt="ba-behaviours" src="{{ '/img/ba-and-beyond-2019/ba-behaviours.png' | prepend: site.baseurl }}" class="image fit" style="margin:0px auto; max-width: 500px;">
+
+## The 7 hidden layers behind agile techniques: Ever wondered why we use post-its during retrospectives? – Pieter Van Driessche
+
+<span class="image left"><img class="p-image" alt="Pieter Van Driessche" src="/img/ba-and-beyond-2019/pieter-van-driessche.jpg"></span>
+[Pieter Van Driessche](https://www.linkedin.com/in/pietervandriessche/){:target="_blank" rel="noopener noreferrer"} has worked in IT for over 20 years and did most of the common software jobs: developer, analyst, team lead and project manager. 
+He experimented with cultural change management during 2 years, after which he returned to IT because of the faster speed of change. 
```

Comment:
```suggestion
He experimented with cultural change management for 2 years, after which he returned to IT because of the faster speed of change. 
```